lib/core/presentation/widgets/event/rsvp_event_success_popup_widget.dart

```diff
@@ -0,0 +1,169 @@
+import 'package:app/core/domain/event/entities/event.dart';
+import 'package:app/core/presentation/widgets/animation/ripple_animation.dart';
+import 'package:app/core/presentation/widgets/back_button_widget.dart';
+import 'package:app/core/presentation/widgets/common/button/linear_gradient_button_widget.dart';
+import 'package:app/gen/assets.gen.dart';
+import 'package:app/gen/fonts.gen.dart';
+import 'package:app/i18n/i18n.g.dart';
+import 'package:app/theme/sizing.dart';
+import 'package:app/theme/spacing.dart';
+import 'package:app/theme/typo.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_screenutil/flutter_screenutil.dart';
+
+class RSVPEventSuccessPopupWidget extends StatelessWidget {
+  const RSVPEventSuccessPopupWidget({
```

Comment:
Since it is a full screen widget (Scaffold), refactor it's name to `page` to avoid misunderstanding.

Reply:
👍